src/editors/data/redux/video/selectors.js

```diff
@@ -23,6 +23,38 @@ export const simpleSelectors = [
   stateKeys.licenseDetails,
 ].reduce((obj, key) => ({ ...obj, [key]: state => state.video[key] }), {});
 
+export const videoSettings = createSelector(
```

Comment:
We'll need to, after we merge transcripts, refactor this

Reply:
The idea of that selector would be to grab all the values we would want to send. I would want to at least re-evaluate it after we finish all the widgets for video editor since it seems like we are changing the data shape as we learn new things about video.